app/assets/javascripts/components/ops/tenant-component.js

```diff
@@ -0,0 +1,94 @@
+ManageIQ.angular.app.component('tenantComponent', {
+  bindings: {
+    recordId: '=?',
```

Comment:
Aah... it works just fine for editing an existing tenant, but does that thing for a new one...

That's beacuse with `@?` the default is `""` and not `undefined`.

The only thing you'd need to change is to to `if (vm.recordId) {` instead of `if (angular.isDefined(vm.recordId)) {` :).